pkg/provider/cloud/packet/provider.go

```diff
@@ -120,6 +120,7 @@ func GetCredentialsForCluster(cloudSpec kubermaticv1.CloudSpec, secretKeySelecto
 
 func ValidateCredentials(apiKey, projectID string) error {
 	client := packngo.NewClientWithAuth("kubermatic", apiKey, nil)
+	client.UserAgent = fmt.Sprintf("kubermatic %s", client.UserAgent)
```

Comment:
I would have plumbed the kubermatic version in if it were obvious how to do so. Version is sent to some, but not all, of the cmds/. It wasn't clear to me if this client was only used in the installer cmd. If so, it may make sense to change this to `kubermatic-installer`, which is used elsewhere in this repo as `userAgent`.

Reply:
`ValidateCredentials` is not used anywhere in our code so it doesn't make too much sense to add this here. Let me check what can be done here.

Regarding version, it should be possible to propagate it down further. Let me check. This code is not used in the installer so `kubermatic` makes more sense here.

Reply:
As per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent, can we not use a space, but a `/` as the separator between product and version?

Reply:
Slash would be used to denote the version of Kubermatic. The space is used to identify the product chain. The default packngo User-Agent is "packngo/{ver}". 

The combined UA in this case: "kubermatic packngo/{ver}" 

In the Mozilla docs, there are examples of Firefox and Mozilla in a chain, space separated. 

The order follows the pattern of other integrations, both in Kubermatic (see the Linode  UA) and packngo (terraform-provider-equinix).

Reply:
@displague I tried pushing in your branch, but it seems I don't have access. Bringing in the version in the User-Agent header might be a bit more complicated than I thought, but I think we can still do it. 

Maybe it's better if I do it in a follow-up PR. This was the commit that I was trying to push https://github.com/kubermatic/kubermatic/commit/99093a0687b94596601bb03dc89c17730f234f67